src/shared/modules/experimentalFeatures/experimentalFeaturesDuck.ts

```diff
@@ -9,13 +9,20 @@ export const showFeature = (state: any, name: any) =>
   !!(state[NAME][name] || {}).on
 
 export const experimentalFeatureSelfName = 'showSelf'
+export const experimentalFeaturePreviewName = 'advertisePreview'
 
 export const initialState = {
   [experimentalFeatureSelfName]: {
     name: experimentalFeatureSelfName,
     on: true,
     displayName: 'Show experimental features',
     tooltip: 'Show feature section in settings drawer'
+  },
+  [experimentalFeaturePreviewName]: {
+    name: experimentalFeaturePreviewName,
+    on: false,
+    displayName: 'Advertise preview of new browser',
+    tooltip: 'Enable the advertisement tile of the new browser'
   }
 }
```

Comment:
I think this is a user facing feature flag, I think we're better off to just have it on always if `isPreviewAvailable`

Reply:
Removed it!